data-reporting-service/person-service/src/test/java/gov/cdc/etldatapipeline/person/DebeziumChangeDataParserTests.java

```diff
@@ -24,8 +24,11 @@ public void parseDebeziumValueTest() {
                 readFileData("rawDataFiles/person/PersonPatientChangeData.json"),
                 "/payload/after",
                 Person.class);
-        Assertions.assertEquals("9005400", person.getPersonUid());
-        Assertions.assertEquals(1708702633619L, person.getTs_ms());
-        Assertions.assertEquals("u", person.getOp());
+        assertEquals("9005400", person.getPersonUid());
+        assertEquals("ProviderUpdate", person.getAddReasonCd());
+        assertEquals("PAT", person.getCd());
```

Comment:
Similar to organization we can consider removing all other fields except for the UID from the debezium change capture data. (if that will improve the code coverage)

Reply:
> Similar to organization we can consider removing all other fields except for the UID from the debezium change capture data. (if that will improve the code coverage)

I totally agree, actually there are two fields from the Person entity used for message processing: `personUid` and `cd`, so all other fields could be removed. What about payload for `nbs_Person`, is there a config which controls all those fields, or we can just ignore it for now?

Reply:
Yeah just `personUid` and `cd` is sufficient. The payload for the `nbs_Person` topic is the same as the debezium change data. It will be the data with `personUid` and `cd` .  
We will be getting all the person data in the debezium change data but deserializing only the above 2 fields.

Reply:
OK, so lets shrink it in the context of this ticket, no separate one is needed then.